mono/utils/checked-build.h

```diff
@@ -201,10 +201,14 @@ void check_metadata_store_local(void *from, void *to);
 
 void checked_build_thread_transition(const char *transition, void *info, int from_state, int suspend_count, int next_state, int suspend_count_delta);
 
+void g_error_with_history(const char *msg, ...);
```

Comment:
I could make it `mono_`… pretty much was just trying to emphasize the symmetry with `g_error`